tests/getComponent.spec.ts

```diff
@@ -48,7 +48,7 @@ describe('getComponent', () => {
   })
 
   it('should throw if not found with a component selector that has no name', () => {
-    const wrapper = mount(compA)
```

Comment:
This test is not relevant anymore, is it? It was testing for the missing behavior which you implemented.

Reply:
It is still relevant - we still should `throw` getComponent if we were unable to find component without name. It just updated to reflect changes in this PR - previously we were searching for compA inside compA - and it throws (not the case anymore), now we search for compA inside compB and it also throws (which is what this test about - about throwing, not about matching root)